new/LIT7083QeneTadewewotu.xml

```diff
@@ -0,0 +1,89 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7083QeneTadewewotu" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ተጼውዎቱ፡ ለ<choice><sic>ስ</sic><corr resp="JK">ሰ</corr></choice>ብእ፡ እመ፡ ከመ፡ ጽዮን፡ ኮነ፡ ምህርካ፡ ጽኑእ፡ እም፡ ተመነነ፤</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Taṣewǝwotu la-sabʾ ʾǝmma kama Ṣǝyon kona mǝhǝrkā ṣənuʾ ʾǝm tamannana (Qǝne of unknown type)</title>
```

Comment:
I would add mawaddǝs-type because of the eight lines. If you are not certain about this, you can add "possibly" or "probably".